README.md

```diff
@@ -1,35 +1,35 @@
 # TEAM ZOMBIE MINECRAFT MOD / COMP5541 Winter 2022
 
-## MOD DESCRIPTION & INFORMATION
+## Description
 
-The idea of this mod is to bring into the world the items and functionalities of the Pokemon world. Items like pokemon balls and functionality like catching and releasing pokemon. 
+The idea of this mod is to implement some items and functionalities of the Pokémon game. These include items such as Poké Balls, and functionalities such as catching and releasing Pokémons.
 
+## Background
 
-### MINECRAFT
+This project is bootstrapped from Forge's MDK 1.18.1-39.0.59, which can be found [here](https://adfoc.us/serve/sitelinks/?id=271228&url=https://maven.minecraftforge.net/net/minecraftforge/forge/1.18.1-39.0.59/forge-1.18.1-39.0.59-mdk.zip).
```

Comment:
Agreed this is fine. I don't believe the installation of forge is required only the use of mdk. Forge is for running distributed mods i believe.

Reply:
Yap!